setup.py

```diff
@@ -20,27 +20,27 @@
 elastic_deps = ["elasticsearch-dsl~=6.4,<7.0"]
 mongo_deps = ["pymongo~=3.11", "mongomock~=3.23"]
 server_deps = [
-    "uvicorn~=0.13.4",
+    "uvicorn~=0.14",
```

Comment:
```suggestion
    "uvicorn~=0.14.0",
```

Reply:
Any reason to pin this so aggressively?

Reply:
(I dropped the patch version on purpose)

Reply:
I know, but following semantic versioning the last one here is not a patch, but a minor version (since the "major" version is 0). This is similar to how we do it for other dependencies with a "major" version of 0.

Reply:
Fair enough! Didn't realize that was the distinction we are making here